source/_integrations/openuv.markdown

```diff
@@ -57,89 +42,36 @@ file:
 ```yaml
 openuv:
   api_key: YOUR_OPENUV_API_KEY
+  elevation: 1609.3
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.